### ISSUE TITLE
Application Crash

### ISSUE THREAD

**Issue by deathgaze** [avocode/avocode#826]
Occurred while opening a Sketch file inside an Avocode project. 
-----
App v.2.6.0
Source UID: 62360
```
Error: ENOENT: no such file or directory, lstat '/Users/ksefchik/Downloads/Avocode.app'
  at Error (native)
  at fs.lstatSync (fs.js:838:18)
  at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:170:16)
  at realpathSync (fs.js:1447:21)
  at Object.fs.realpathSync (ATOM_SHELL_ASAR.js:253:24)
  at tryFile (module.js:118:15)
  at tryExtensions (module.js:126:20)
  at Function.Module._findPath (module.js:166:20)
  at Module._resolveFilename (module.js:322:25)
  at Function.Module._resolveFilename (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:266:25)
  at Module.require (module.js:353:17)
  at require (module.js:372:17)
  at a.module.exports.a.getVariables (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/language-manager.js:1:2943)
  at a.module.exports.a.processLayers (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/language-manager.js:1:3428)
  at Object.module.exports._processLayers (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/code-panel/lib/main.js:1:3738)
  at Object.module.exports._onVariablesChange (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/code-panel/lib/main.js:1:3380)
  at emitNone (events.js:72:20)
  at emit (events.js:163:7)
  at e.module.exports.e.emitChange (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/stores/base.js:1:558)
  at e.setItems (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/stores/variables.js:1:1518)
  at t.<anonymous> (/Users/ksefchik/Downloads/Avocode.app/Contents/Resources/app.asar/
```

**Comment by vojtatranta** [comment]
We are sorry for this issue :(
But here is a fix! #534 
Just be sure to move Avocode.app from downloads folder to the Applications folder on your Mac.
Because it seems you have deleted it from downloads directory.
Let us know whether it worked and sorry again :).

**Comment by connor-bean** [comment]
Hey!

We have released the newest version of the Avocode app, version 2.8, which we expect to fix this problem. Please download it at https://avocode.com/download/. I am going to close this ticket but please reopen it if you’re still experiencing this problem in Version 2.8.

Thanks!
Connor, Avocode Support
